src/hashtags.ts

```diff
@@ -149,6 +158,57 @@ export = class HashtagPlugin extends DB2Plugin {
     }
   }
 
+  /**
+   * Generalized version of `getMesssagesByHashtag`.
+   * Gets the `seq` of all messages that have *some* hashtag.
+   */
+  getMessagesWithSomeHashtag(
+    live: 'liveOnly' | 'liveAndOld' | undefined,
+    cb: (err: any, opData?: any) => void,
+    onAbort: (listener: () => void) => void,
+  ) {
+    if (live === 'liveAndOld') return cb(new Error('unimplemented liveAndOld'));
+
+    const sharedReadOpts = {
+      gt: ['', ''],
+      lt: [undefined, undefined],
```

Comment:
is this the proper way to read everything in the index? seems to work but wasn't sure of the proper values to use here

Reply:
It's weird, but yes I think it's the proper way. :upside_down_face: